core/storage/skill/gae_models.py

```diff
@@ -77,22 +77,21 @@ def _trusted_commit(
                     cmd: str. Unique command.
                 and then additional arguments for that command.
         """
-        super(SkillModel, self)._trusted_commit(
-            committer_id, commit_type, commit_message, commit_cmds)
-
         committer_user_settings_model = (
             user_models.UserSettingsModel.get_by_id(committer_id))
         committer_username = (
             committer_user_settings_model.username
             if committer_user_settings_model else '')
 
         skill_commit_log_entry = SkillCommitLogEntryModel.create(
-            self.id, self.version, committer_id, committer_username,
+            self.id, self.version + 1, committer_id, committer_username,
```

Comment:
See my notes in the other PR about finding the root cause of the issue that necessitates this change.

Reply:
Done, changed to put().